backend/seeds/00200_nativeLanguages.js

```diff
@@ -5,6 +5,7 @@ const SEED_NATIVELANGUAGES = [
   {NativeLanguage: 'TAMIL'},
   {NativeLanguage: 'MALAY'},
   {NativeLanguage: 'BANGLA'},
+  {NativeLanguage: 'BENGALI'},
```

Comment:
hi ying tong! Bengali and Bangla refer to the same language. Bengali is usually referred to as Bangla among Bangladeshi people (i volunteer with TWC2 and a lot of guys refer to bengali as bangla, wikipedia also says so here https://en.wikipedia.org/wiki/Bengali_language)

Reply:
thanks! i've removed this diff.